Tests/MapboxMapsTests/Foundation/Camera/CameraAnimatorMocks.swift

```diff
@@ -0,0 +1,21 @@
+import XCTest
+
+#if canImport(MapboxMaps)
+@testable import MapboxMaps
+#else
+@testable import MapboxMapsFoundation
+#endif
+
+public class CameraAnimatorMock: CameraAnimatorDelegate {
```

Comment:
Look into using https://github.com/mapbox/mapbox-maps-ios/blob/main/Tests/MapboxMapsTests/Helpers/Stub.swift instead of creating this.